src/Nox.Types/Types/Time/Time.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Globalization;
+
+namespace Nox.Types;
+
+/// <summary>
+/// Represents a time of day, as would be read from a clock, within the range 00:00:00 to 23:59:59.9999999.
+/// </summary>
+public sealed class Time : ValueObject<DateTime, Time> // This should be TimeOnly
```

Comment:
is there a reason to not be TimeOnly Struct? for the underline type?

Reply:
TimeOnly is not available as part of .Net standard 2.0 framework.

Reply:
Can you copy this class and see if it compiles without an extra depdendency https://github.com/dotnet/dotnet/blob/5b09052ac0948cc1a94829f01491c550d4837757/src/runtime/src/libraries/System.Private.CoreLib/src/System/TimeOnly.cs#L4 ?

I'm taking care of migrating our pacakages to a more recent version